crm-plugin/services/utils.js

```diff
@@ -0,0 +1,18 @@
+const { isEmpty } = require("lodash");
+
+const checkParams = (values, requiredValues) => {
+  //Checks if params are empty
+  if (isEmpty(values)) {
+    return "Params are empty";
```

Comment:
dont return plain message this way.return json object below:
In case of error below:
{"error":true , "mesgage":"error message here"}
no error below:
{"error":false , "mesgage":""}